packages/api-gateway/src/drivers/http/adapters/usuarios/handlers.js

```diff
@@ -68,13 +68,36 @@ async function createUsuario(req, reply) {
 
 async function updateUsuario(req, reply) {
   try {
+    Logger.info('[usuarios]: Updating usuario');
+
     const { usuarioId } = req.params;
-    const { body } = req;
+    const { fotoPerfil, ...data } = req.body;
 
-    Logger.info('[usuarios]: Updating usuario');
-    const usuarioUpdated = await this.usuarioServices.updateUsuario(
+    let usuarioUpdated = await this.usuarioServices.updateUsuario(
+      usuarioId,
+      data,
+    );
+
+    const files = [];
+
+    if (fotoPerfil) {
+      if (usuarioUpdated.personaId) {
```

Comment:
Make this a single if
```suggestion
    if (fotoPerfil && usuarioUpdated.personaId) {
```